src/state/Reducer.test.jsx

```diff
@@ -0,0 +1,51 @@
+import reducer from './Reducer';
```

Comment:
It seems this test doesn't run when executing `npm test`:

![image](https://user-images.githubusercontent.com/124746/115485073-9d304300-a219-11eb-9112-59065aed5281.png)